dlio_benchmark/utils/config.py

```diff
@@ -272,7 +272,7 @@ def derive_configurations(self, file_list_train=None, file_list_eval=None):
         if self.data_loader_sampler is None and self.data_loader_classname is None:
             if self.data_loader == DataLoaderType.TENSORFLOW:
                 self.data_loader_sampler = DataLoaderSampler.ITERATIVE
-            elif self.data_loader in [DataLoaderType.PYTORCH, DataLoaderType.DALI]:
+            elif self.data_loader in [DataLoaderType.PYTORCH, DataLoaderType.PYTORCH_DIRECT, DataLoaderType.DALI]:
```

Comment:
Again, this shows it has nothing to do with PYTORCH. Lets fix the overall design here.